### PR TITLE
Create /AppContext directory in context/ to store there current files

### PR DESCRIPTION
**Pull request by LinconOssa**
The AppContext directory is created to store index.tsx and types.ts and everything still works the same

### CODE REVIEW

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/home/interface.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useContext } from "react";
 import { Header, Stack, Nav, useMediaQueries } from "@inube/design-system";
 
-import { AppContext } from "@src/context";
+import { AppContext } from "../../context/AppContext";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/home/interface.tsx]: image
Changed

File: src/pages/home/Home.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Home } from "./index";
 import { StoryFn } from "@storybook/react";
 import { BrowserRouter } from "react-router-dom";
-import AppContextProvider from "../../context/index";
+import AppContextProvider from "../../context/AppContext/index";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/home/Home.stories.tsx]: image
Changed

File: src/App.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import {
 import { ErrorPage } from "@components/layout/ErrorPage";
 import { Home } from "@pages/home";
 import { GlobalStyles } from "@styles/global";
-import AppContextProvider from "./context";
+import AppContextProvider from "./context/AppContext";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/App.tsx]: image
Changed

File: src/context/AppContext/index.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { createContext } from "react";
-import linparLogo from "../assets/images/linpar.png";
+import linparLogo from "../../assets/images/linpar.png";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/context/AppContext/index.tsx]: image
I changed it to @assets

**Review by carin91 — APPROVED** [review]

**Review by cmarin001 — APPROVED** [review]
meet the requirements

**Pull request merged**